packages/twenty-front/src/modules/settings/roles/hooks/useHasSettingsPermission.ts

```diff
@@ -0,0 +1,22 @@
+import { currentUserWorkspaceState } from '@/auth/states/currentUserWorkspaceState';
+import { useRecoilValue } from 'recoil';
+import { SettingsFeatures } from 'twenty-shared';
+
+export const useHasSettingsPermission = (
```

Comment:
```suggestion
export const useHasSettingsPermissionOnSettingsFeature = (
```
?

I find notions of settings permission / settings feature are a bit mixed (probably my fault originally!)

Reply:
Yes I agree! Let's use Permissions instead of Feature for the enum type, I was originally using "feature" but found it confusing when mixing with feature flag logic @ijreilly 🙂